jbpm-container-test/integration-deps/issue-keeper/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>integration-deps</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>idep-issue-keeper</artifactId>
+
+  <name>Integration Dependencies :: Issue Keeper</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>link.bek.tools</groupId>
```

Comment:
I think it is. Sometimes the issue is not so severe that it needs a fix immediately. Or it takes some time to fix it and there are more important tasks to do. It is nice to have Issue Keeper in place instead of Ignore annotations. We already use it in [jbpm-test-coverage](https://github.com/droolsjbpm/jbpm/tree/master/jbpm-test-coverage) or [kie-wb-tests-rest](https://github.com/droolsjbpm/kie-wb-distributions/tree/master/kie-wb-tests/kie-wb-tests-rest).